x-pack/filebeat/input/netflow/decoder/v9/lru.go

```diff
@@ -82,7 +95,11 @@ func (h *pendingTemplatesCache) Add(key SessionKey, events *bytes.Buffer) {
 	h.mtx.Lock()
 	defer h.mtx.Unlock()
 
-	h.events[key] = append(h.events[key], events)
+	q := h.events[key]
+	if len(q) >= maxPendingPerKey || h.totalPendingLocked() >= maxTotalPending {
+		return
```

Comment:
I think we should log something here - this drops the event?

Reply:
IMO, we can avoid logging here. As we are not dropping event we are just not adding entry in cache when it's already full.

Reply:
there is no infrastructure to log here. the cache drops events during its cleanup interval. this is no different. we don't want to fill up the cache needlessly.